integrations/evm/src/runtime.ts

```diff
@@ -0,0 +1,156 @@
+import { DataItem, IRuntime, Validator, VOTE } from "@kyvejs/protocol";
+import { name, version } from "../package.json";
+import { providers, utils } from "ethers";
+import { TransactionReceipt } from "@ethersproject/abstract-provider";
+import { createHashesFromBundle, generateMerkleRoot } from "../utils/merkle";
+
+// EVM config
+interface IConfig {
+  rpc: string;
+  finality: number;
+  includedData: {
+    blockWithTransactions: boolean,
+    blockReceipts: boolean,
+    transactionReceipts: boolean,
+  }
+}
+
+export default class EVM implements IRuntime {
+  public name = name;
+  public version = version;
+  public config!: IConfig;
+
+  async validateSetConfig(rawConfig: string): Promise<void> {
+    const config: IConfig = JSON.parse(rawConfig);
+
+    if (!config.rpc) {
+      throw new Error(`Config does not have property "rpc" defined`);
+    }
+
+    if (process.env.KYVEJS_EVM_RPC) {
+      config.rpc = process.env.KYVEJS_EVM_RPC;
+    }
+
+    if (!config.finality) {
+      throw new Error(`Config does not have finality defined`);
+    }
+
+    if (!config.includedData.blockWithTransactions && !config.includedData.blockReceipts && !config.includedData.transactionReceipts) {
```

Comment:
check if includedData exists, if it is null this line will throw an error. You can do this with this `!config.includedData?.blockWithTransactions ...`